### PR TITLE
Namedtuple test + cleanup

### PR DESCRIPTION
**Pull request by ygravrand**
To increase our coverage a little bit.

### CODE REVIEW

**Comment by ygravrand** [comment]
Fails on Pypy... interesting!

File: tests/cloudpickle_test.py [hunk 1]
```diff
@@ -60,26 +67,20 @@ def __getattr__(self, item):
...
-        getter2 = pickle.loads(cloudpickle.dumps(getter))
+        getter2 = pickle_depickle(getter)
         self.assertEqual(getter(d), getter2(d))
 
-    # Regression test for SPARK-3415
-    def test_pickling_file_handles(self):
-        out1 = sys.stderr
-        out2 = pickle.loads(cloudpickle.dumps(out1))
-        self.assertEquals(out1, out2)
-
```
Comment (ogrisel) [inline tests/cloudpickle_test.py]: Why did you remove this one? It seems unrelated to the purpose of that PR. Is it already tested somewhere else?

**Comment by ygravrand** [comment]
Yes, already tested in cloudpickle_file_test:test_pickling_special_file_handles

**Comment by ogrisel** [comment]
PyPy default pickle supports named tuples:

```python
>>>> from collections import namedtuple
>>>> P = namedtuple("P", "x y")
>>>> import pickle
>>>> nt_pickled = pickle.dumps(P(1, 2))
>>>> pickle.loads(nt_pickled)
P(x=1, y=2)
```

so this is a pbm in cloudpickle.

**Comment by rgbkrk** [comment]
I'm assuming we can probably close this one for now?

**Comment by ogrisel** [comment]
Well the test failure highlight a bug in cloudpickle for namedtuple with PyPy.

**Comment by rgbkrk** [comment]
Oh whoop. Should we bring this in as a skip test and make an issue?

**Comment by rgbkrk** [comment]
I'm going to go ahead and close this. If you come back at it @ygravrand, feel free to re-open or create a new PR.
